data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	Both fall to the netting below, but Pairri clearly hits first. You hear the whistle once again, and after they are helped back to the platform, they approach each other and place one of their frontal legs over the other's. They give what must be the Kimek equivalent of a handshake as it is announced that Pairri's opponent has won.`
+			`	With the event finally over, the audience begins to return to the elevators, with Heliarch agents making sure everyone proceeds down and back to the entrance in an orderly fashion. There, you see a second train in addition to the one you arrived on, and the Arach who helped explain the event to you earlier informs you that it leads to where the cooperative game will take place. The first will also take you back to where you can observe the other games.`
+				goto choice
+			
+			
+			label arachi
+			apply
+				set "coalition games: arachi"
+			`	When the train stops, you find yourself inside a building similar to a spaceport, but the ships are more akin to a mix between a helicopter and a raft. As other visitors start boarding the flying platforms, you follow suit and find that each one is filled by dozens of adjustable seats. It seems these seats' default position is designed for Arachi, but moving a few parts makes them capable of easily accommodating Kimek and even Saryds. After fiddling with yours for a while, you manage to adjust it to a configuration which is suitable enough, inasmuch as it is more comfortable than the others.`
+			`	Once the rafts are full, they leave their docks and depart the port. Leaving the building, you realize it was built into the side of a highland, and you're immediately soaring out over a lush rainforest. Increasingly larger trees emerge from the carpet of green below, which seems to shimmer with the movement of the various animals in the canopy. Bird-like creatures with funneled beaks, startled by the rafts' engines, appear from the large flocks and soar ahead of and away from the fleet of flying rafts before settling back into the trees. In the distance, there is a river so wide you at first believe it to be an ocean, until a Heliarch agent informs you otherwise.`
```

Comment:
```suggestion
			`	Once the rafts are full, they leave their docks and depart the port. Leaving the building, you realize it was built into the side of a highland, and you're immediately soaring out over a lush rainforest. Increasingly larger trees emerge from the carpet of green below, which seems to shimmer with the movement of the various animals in the canopy. Bird-like creatures with funneled beaks, startled by the rafts' engines, appear in large flocks and soar ahead of and away from the fleet of flying rafts before settling back into the trees. In the distance, there is a river so wide you at first believe it to be an ocean, until a Heliarch agent informs you otherwise.`
```